src/pages/ReimbursementAccount/BankAccountStep.js

```diff
@@ -274,36 +230,30 @@ class BankAccountStep extends React.Component {
                     </ScrollView>
                 )}
                 {subStep === CONST.BANK_ACCOUNT.SETUP_TYPE.PLAID && (
-                    <FormScrollView>
-                        <View style={[styles.mh5, styles.mb5, styles.flex1]}>
-                            <AddPlaidBankAccount
-                                text={this.props.translate('bankAccount.plaidBodyCopy')}
-                                onSelect={(params) => {
-                                    this.setState({
-                                        selectedPlaidBankAccount: params.selectedPlaidBankAccount,
-                                    });
-                                }}
-                                onExitPlaid={() => BankAccounts.setBankAccountSubStep(null)}
-                                receivedRedirectURI={this.props.receivedRedirectURI}
-                                plaidLinkOAuthToken={this.props.plaidLinkOAuthToken}
-                                allowDebit
-                            />
-                        </View>
-                        {!_.isUndefined(this.state.selectedPlaidBankAccount) && (
-                            <FormAlertWithSubmitButton
-                                isAlertVisible={Boolean(error)}
-                                buttonText={this.props.translate('common.saveAndContinue')}
-                                onSubmit={this.addPlaidAccount}
-                                message={error}
-                                isLoading={loading}
-                            />
-                        )}
-                    </FormScrollView>
+                    <Form
+                        formID={ONYXKEYS.REIMBURSEMENT_ACCOUNT}
+                        validate={this.validatePlaidAccount}
+                        onSubmit={this.addPlaidAccount}
+                        submitButtonText={this.props.translate('common.saveAndContinue')}
+                        style={[styles.mh5, styles.mb5, styles.flex1]}
```

Comment:
This is adding an additional margin at the bottom of the submit button. I think this should be the same as the Form in `subStep === CONST.BANK_ACCOUNT.SETUP_TYPE.MANUAL` 
```suggestion
                        style={[styles.mh5, styles.flexGrow1]}
```

Reply:
@luacmartins can you please help me in testing the plaid account, how can I add a test plaid account?
right now plaid is being opened in a new tab with the real bank account site

Reply:
@jayeshmangwani First step is enable Settings > Preferences > Use Secure staging and then follow these steps after clicking on 'Connect online with Plaid':

- Click "Log into your bank"
- Plaid modal - Choose Wells Fargo (use Bank of America if Wells Fargo is not available)
- Plaid modal - User Id: user_good
- Plaid modal - Password: pass_good
- Plaid modal - Submit
- Plaid modal - If they request a verification code via mobile sms, enter: credential_good when required
- Choose an account: Plaid Saving11122XXXXXX111
- Save & Continue

Reply:
Thanks, @youssef-lr 
I have tried these steps, but still getting invalid credentials 
attaching a video of the steps that I am following https://drive.google.com/file/d/1BXJ_w8mTqnuUJExWQAToGdwUEgtU138W/view?usp=sharing

Reply:
It doesn't look like you're hit the staging API, you should see at the bottom "You are currently in Sandbox mode", which is why the credentials fail. I'm not sure exactly why it's not working for you, can you try running `npm run web` again?
<img width="776" alt="Screen Shot 2022-09-19 at 21 47 51" src="https://user-images.githubusercontent.com/9680864/191113700-633cc48b-e827-4dbb-b915-65656a743e8d.png">

Reply:
Sure, let me try app again

Reply:
@jayeshmangwani You could even try and copy the `.env.staging` to `.env` to connect to the staging API and then run via Desktop App.

Reply:
Thank you @mananjadhav @youssef-lr now I am to run Sandbox mode,

@luacmartins suggested change has been done 
https://github.com/Expensify/App/pull/10900#discussion_r974588849

added `isSubmitButtonVisible` prop to form

and one more thing is that I have removed inputID from AddPlaidBankAccount component in BankAccountStep as we are setting selectedPlaidBankAccount state value on the BankAccountStep page

Reply:
Thanks for the changes! I'll re-review this tomorrow.

Reply:
@youssef-lr both suggested changes are done, and I have removed inputID changes from AddPlaidBankAccount that I had added and I think that should not be needed for this PR
Thanks.